tools/nvqpp/nvq++.in

```diff
@@ -11,11 +11,16 @@
 # This is a skeleton driver script to compile C++ source that contains
 # CUDA Quantum code.
 
-RED=$(tput setaf 1)
-NORMAL=$(tput sgr0)
 function error_exit {
 	callerName=$(basename "$(caller 0)")
-	echo "${RED}${callerName}:$(echo $(caller 0) | cut -d " " -f1): $1${NORMAL}"
+	message_content="${callerName}:$(echo $(caller 0) | cut -d " " -f1): $1"
```

Comment:
Back to black and white. 😃

Reply:
Does this show in black-and-white for you? because this should work both with and without tput. Double checked that message_content="hello" is red for both cond=true and cond=false in
```
if $cond; then          
    RED=$(tput setaf 1);            
    NORMAL=$(tput sgr0);           
    echo "${RED}${message_content}${NORMAL}";       
else        
    echo -e "\e[01;31m${message_content}\e[0m" >&2;    
fi
```